Screens/PerformanceScreen.js

```diff
@@ -75,15 +74,13 @@ class PerformanceScreen extends LocalizeComponent {
                                 <Text style={styles.textSubtitle}>{this.t("description")}</Text>
                                 <Text style={styles.testStyle}>{this.props.performance.description}</Text>
                                 <Text style={styles.textSubtitle}>{this.t("price")}</Text>
-                                <Text style={styles.testStyle}>{this.props.performance.minPrice} - {this.props.performance.maxPrice} {this.t("uah")}</Text>
+                                <Text style={styles.testStyle}>{this.props.performance.price == 0 ? this.props.performance.minPrice + "-" + this.props.performance.maxPrice : this.props.performance.price} {this.t("uah")}</Text>
```

Comment:
use template strings to build string https://developers.google.com/web/updates/2015/01/ES6-Template-Strings